app/views/course/assessment/question/scribing/new.html.slim

```diff
@@ -1,3 +1,4 @@
 - add_breadcrumb :new
+- template = 'course/assessment/question/scribing/scribing_question.json.jbuilder'
 = page_header
-div#scribing-question
+div#scribing-question data=(render partial: template)
```

Comment:
Actually doesn't this already render the jbuilder partial that includes the skills? So you don't have to make another call to the server?